python/sglang/srt/models/deepseek_v2.py

```diff
@@ -1932,6 +1932,8 @@ def post_load_weights(self, is_nextn=False, weight_names=None):
             self._weight_requant_ue8m0()
 
     def _weight_requant_ue8m0(self):
+        if self.config.architectures[0] == "DeepseekV3ForCausalLMNextN":
```

Comment:
hmm, maybe the MTP layer also have weights that needs to be requantized

thus I guess maybe change `for layer_id in ...` in the `_weight_requant_ue8m0` into something that allows to handle mtp layer

Reply:
I see. Do you mean this will affect the accept length?

Reply:
Yes my naive guess is that, it will make nextn layer to have wrong output. But have not do any experiments and I can be totally wrong